astropy_helpers/tests/test_setup_helpers.py

```diff
@@ -10,6 +10,19 @@
 from . import *
 
 
+def run_setup_patched(*args, **kwargs):
+    """
+    In Python 3, on MacOS X, the import cache has to be invalidated otherwise
+    new extensions built with ``run_setup`` do not always get picked up.
+    """
+    try:
+        return run_setup(*args, **kwargs)
+    finally:
+        if sys.version_info[0] >= 3:
+            import importlib
+            importlib.invalidate_caches()
```

Comment:
Oh, my thinking was just to name this `run_setup` so that it would be transparent (I've had reasons to wrap `run_setup` in the past too, but have since removed them since they weren't necessary anymore, but now there's a new reason again.  I'll just update it myself.